binary/make/makefile

```
@@ -3,6 +3,6 @@
 # goal of QUIET is to make it easier to spot warnings and/or circular dependency
 # errors.
 
-QUIET = yes
+QUIET = no
```

I think we want `yes` in production.
```suggestion
QUIET = yes
```